src/utils/PublicIdentity/Encoding.ts

```diff
@@ -0,0 +1,19 @@
+import { IPublicIdentity } from '@kiltprotocol/sdk-js'
+
+type TPublicIdentityEncoded = {
+  a: string
+  b: string
+  s?: string
+}
+
+const encodePublicIdentity = (
+  publicIdentity: IPublicIdentity
+): TPublicIdentityEncoded => ({
+  a: publicIdentity.address,
+  b: publicIdentity.boxPublicKeyAsHex,
```

Comment:
If you want to have it even smaller, you could convert it to base64 first

Reply:
Good idea; for now I'm sticking to a simple array.